src/mango/test/02-basic-find-test.py

```diff
@@ -222,6 +222,20 @@ def test_sort(self):
         docs2 = list(reversed(sorted(docs1, key=lambda d: d["age"])))
         assert docs1 is not docs2 and docs1 == docs2
 
+    def test_sort_desc_complex(self):
```

Comment:
Could you add a test without the `manager: $exists: true` and make sure it returns an error